recipe-client-addon/lib/PreferenceManager.jsm

```diff
@@ -106,19 +109,19 @@ this.PreferenceManagement = (namespace) => ({
   withMockData(testFunction) {
     // We use a POJO to fake the internal storage.
     this.mockData = {
-      [namespace]: {},
+      [this.namespace]: {}
     };
 
     return async (...args) => {
       this.storePromise = Promise.resolve({
         data: this.mockData,
-        saveSoon() {},
+        saveSoon() {}
```

Comment:
Nit: trailing comma was correct here and 6 lines up, unless eslint told you otherwise? Convention in Firefox code is usually to keep it, though I think devtools might have a different one...